mcc/health/cmd/create.go

```diff
@@ -150,6 +165,13 @@ func init() {
 		" ",
 		"Field separator string for output",
 	)
+	createCmd.PersistentFlags().StringVarP(
```

Comment:
What does this do and what are all these params?

Reply:
This adds an flag to the command-line client and stores the value of the argument into the variable `email`. The `PersistentFlags()` allows subcommands of this command to inherit the flag as an option (we don't use it so far) and `StringVarP` implies the option will be stored as a string, and the variable is passed as a pointer to string.

Reply:
so it allows for:
```
health create --email whatever@devex.com
```

Reply:
First param is the pointer to the variable that stores the value. Second is the long name of the variable (`--email`). Third is the short name (none here, but if `e` would allow for `-e`). Fourth is the default value of the flag (none again). Fifth is a description that will be showed in the command's help.

Reply:
Ah, alright – thanks for explaining! :)